packages/driver/src/dom/visibility/fastIsHidden.ts

```diff
@@ -0,0 +1,174 @@
+import $elements from '../elements'
+
+import { unwrap, wrap, isJquery } from '../jquery'
+const { isOption, isOptgroup, isBody, isHTML } = $elements
+
+const DEBUG = false
```

Comment:
guessing this is just a local thing to leverage? can we leverage the debug library to set in the browser? I'm fine with either but maybe a comment